python/ctranslate2/converters/fairseq.py

```diff
@@ -87,11 +89,21 @@ def _load(self):
 
         with torch.no_grad():
             checkpoint = checkpoint_utils.load_checkpoint_to_cpu(self._model_path)
-            args = checkpoint["args"]
+            if checkpoint["args"]:
+                args = checkpoint["args"]
+            else:
+                args = checkpoint["cfg"]["model"]
```

Comment:
Can be simplified to:

```suggestion
            args = checkpoint["args"] or checkpoint["cfg"]["model"]
```